### PR TITLE
NMS-14578: Show resolving UEIs in alarm/detail.jsp

### PR DESCRIPTION
**Pull request by christianpape**
* JIRA: https://opennms.atlassian.net/browse/NMS-14578

### CODE REVIEW

**Comment by RangerRick** [comment]
Should this really only be against 31.x? Is there any reason for doing 31.x-only commits that don't go into `foundation-2023`?

**Comment by christianpape** [comment]
> Should this really only be against 31.x? Is there any reason for doing 31.x-only commits that don't go into `foundation-2023`?

right, rebased to foundation-2023

File: opennms-config-model/src/main/java/org/opennms/netmgt/xml/eventconf/Events.java [hunk 1]
```diff
@@ -69,6 +54,22 @@
...
+import javax.xml.bind.annotation.XmlType;
+
+import org.opennms.core.xml.JaxbUtils;
+import org.opennms.core.xml.ValidateUsing;
+import org.opennms.netmgt.config.utils.ConfigUtils;
+import org.springframework.core.io.DefaultResourceLoader;
+import org.springframework.core.io.Resource;
+import org.springframework.dao.DataAccessResourceFailureException;
+import org.springframework.orm.ObjectRetrievalFailureException;
+import org.springframework.util.StringUtils;
```
Comment (RangerRick) [inline opennms-config-model/src/main/java/org/opennms/netmgt/xml/eventconf/Events.java:71]: the editor wars are at it again!

**Review by RangerRick — APPROVED** [review]
Looks OK to me, although that test failure is odd, doesn't seem like it would be related. I went ahead and reran it, we shouldn't merge until/unless it goes green.

**Pull request merged**